plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2335,3 +2337,28 @@ function wc_update_600_migrate_rate_limit_options() {
 function wc_update_600_db_version() {
 	WC_Install::update_db_version( '6.0.0' );
 }
+
+/**
+ * Create the product attributes lookup table and initiate its filling,
+ * unless the table had been already created manually (via the tools page).
+ *
+ * @return false Always false, since the LookupDataStore class handles all the data filling process.
+ */
+function wc_create_product_attributes_lookup_table() {
```

Comment:
Should we continue the pattern of including the version number in the update function name here?